src/main/java/org/jenkins/tools/test/hook/AbstractMultiParentHook.java

```diff
@@ -87,39 +79,6 @@ public Map<String, Object> action(Map<String, Object> moreInfo) {
         return moreInfo;
     }
 
-    private void cloneFromSCM(
-            File parentPath, String scmTag, String url, String fallbackGitHubOrganization) {
-
-        List<String> connectionURLs = new ArrayList<>();
-        connectionURLs.add(url);
-        if (fallbackGitHubOrganization != null) {
-            connectionURLs =
-                    PluginCompatTester.getFallbackConnectionURL(
-                            connectionURLs, url, fallbackGitHubOrganization);
-        }
-
-        IOException lastException = null;
-        for (String connectionURL : connectionURLs) {
-            if (connectionURL != null) {
-                // See: https://github.blog/2021-09-01-improving-git-protocol-security-github/
-                connectionURL = connectionURL.replace("git://", "https://");
-            }
-            try {
-                PluginCompatTester.clone(connectionURL, scmTag, parentPath);
-                break;
-            } catch (IOException e) {
-                if (lastException != null) {
-                    e.addSuppressed(lastException);
-                }
-                lastException = e;
-            }
-        }
-
-        if (lastException != null) {
-            throw new UncheckedIOException(lastException);
-        }
-    }
-
```

Comment:
Minor cleanup while I was here: deduplicating this code with that in `PluginCompatTester`.